### PR TITLE
LRQA-67892 - Create test stubs for Workflow Metrics - All Items (Transition Tasks)

### PR DESCRIPTION
**Pull request by lucasperj**

### CODE REVIEW

**Comment by anabuchmann** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+	@description = "LPS-108138 - Verify that there are loading cards in Select Steps to Transition and Choose Transition per Step"
+	@ignore = "Test Stub"
+	@priority = "2"
+	test LoadingCardsAreDisplayed {
+
+		// TODO LRQA-67810 LoadingCardsAreDisplayed pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that the message (Your connection has been unexpectedly lost. Select 'Done' to retry.) is displayed when the connection is lost before the user clicks on (Done)"
```
Comment (anabuchmann) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase:67]: Please, change the description for `"LPS-108138 - Verify that the message -Your connection has been unexpectedly lost. Select 'Done' to retry.- is displayed when the connection is lost before the user clicks on Done"`

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+	@ignore = "Test Stub"
+	@priority = "2"
+	test MessageIsDisplayedWhenConnectionIsLost {
+
+		// TODO LRQA-67823 MessageIsDisplayedWhenConnectionIsLost pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that the message (The selected steps have transitioned successfully) is displayed when several steps are transitioned at once"
```
Comment (anabuchmann) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase:76]: Please, change the description for `"LPS-108138 - Verify that the message -The selected steps have transitioned successfully- is displayed when several steps are transitioned at once"`

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+	@ignore = "Test Stub"
+	@priority = "3"
+	test MessageIsDisplayedWhenSeveralStepsAreTransitionedAtOnce {
+
+		// TODO LRQA-67820 MessageIsDisplayedWhenSeveralStepsAreTransitionedAtOnce pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that the message (Select a transition) and (All steps require a transition to be selected to complete this action.) are displayed when the user clicks on Done without selecting all transitions"
```
Comment (anabuchmann) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase:85]: Please, change the description for `"LPS-108138 - Verify that the messages -Select a transition- and -All steps require a transition to be selected to complete this action.- are displayed when the user clicks on Done without selecting all transitions"`

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+	@ignore = "Test Stub"
+	@priority = "3"
+	test MessageIsDisplayedWhenSomeTransitionIsNotSelected {
+
+		// TODO LRQA-67818 MessageIsDisplayedWhenSomeTransitionIsNotSelected pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that, in (Choose Transition Per Step) modal, the user can add comments when transitioning tasks"
```
Comment (anabuchmann) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase:94]: Please, remove the parenthesis from (Choose Transition Per Step).
When, in the test map, you have double quotation marks in the test case description, you can only remove them from the stub description.

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+	@description = "LPS-108138 - Verify that users can filter the tasks to be transitioned by steps"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test OnChooseTransitionPerStepCanFilterByStep {
+
+		// TODO LRQA-67805 OnChooseTransitionPerStepCanFilterByStep pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that only the tasks selected on (Select Steps to Transition) are being considered to be transitioned in the (Choose Transition Per Step) step"
```
Comment (anabuchmann) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase:121]: Please, remove the parenthesis from (Select Steps to Transition).
When, in the test map, you have double quotation marks in the test case description, you can only remove them from the stub description.

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+	@ignore = "Test Stub"
+	@priority = "4"
+	test OnlySelectedTasksAreConsideredToBeTransitioned {
+
+		// TODO LRQA-67811 OnlySelectedTasksAreConsideredToBeTransitioned pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that when there are 11 or more instances to be transitioned with the same step, only the first 10 instances are displayed with the (Show All/Show Less) buttons available to be selected"
```
Comment (anabuchmann) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase:130]: Please, remove the parenthesis from (Show All/Show Less).
When, in the test map, you have double quotation marks in the test case description, you can only remove them from the stub description.

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+	@ignore = "Test Stub"
+	@priority = "3"
+	test ShowAllLessButtonAvailable {
+
+		// TODO LRQA-67802 ShowAllLessButtonAvailable pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that the tasks on (Choose Transition Per Step) are grouped by versioned tasks (when there are changes in the workflow process)"
```
Comment (anabuchmann) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase:139]: Please, remove the parenthesis from (Choose Transition Per Step).
When, in the test map, you have double quotation marks in the test case description, you can only remove them from the stub description.

**Review by anabuchmann — CHANGES_REQUESTED** [review]
Hi @lucasperj  !
Could you, please, check my comments and make the changes that I requested?
Thank you!

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase (hunk 6 above)
Comment (lucasperj) [reply modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase]: @anabuchmann  Done!

**Comment by rodrigocunhaa** [comment]
ci:test:sf

**Comment by anabuchmann** [comment]
ci:test:sf

**Review by anabuchmann — APPROVED** [review]

**Comment by anabuchmann** [comment]
ci:forward

**Comment by rodrigocunhaa** [comment]
Just started reviewing :)

**Pull request closed without merging**